### PR TITLE
[NEW FEATURE][WORK IN PROGRESS]Implement Basics for a Web App

### PR DESCRIPTION
**Pull request by MatejMecka**
Thanks for showing interest in the project and sending us a pull request.<br/>
Please prefix your pull request with [NEW FEATURE], [BUG] or [IMPROVEMENT]

***
__What issue/feature is fixed?__

#99 I am creating a Flask Handler with tests and documentation
***
__Issue/Feature Details__<br/>
You can mention the line of code that is causing the error, detail the feature you are requesting and where it can be useful.

N/A
***
__How To Test It Locally?__<br/>
`cd Nephos/nephos/web`
`python3 webServer.py`

Visit in your browser

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ pytz = "*"
 tzlocal = "*"
 google-api-python-client = "*"
 "oauth2client" = "*"
+"flask" = "*"
```
Comment (thealphadollar) [inline Pipfile]: You should also add flask_testing module in developmental dependencies.

**Comment by MatejMecka** [comment]
screen shot 2018-12-06 at 5 50 59 pm
screen shot 2018-12-06 at 5 50 49 pm
screen shot 2018-12-06 at 5 50 27 pm

Preview of the Web App

File: Pipfile [hunk 2]
```diff
@@ -11,6 +11,7 @@ codecov = "*"
 click = "*"
 scandir = "*"
 "pathlib2" = "*"
+"Flask-Testing" = "*"
```
Comment (thealphadollar) [inline Pipfile]: I still don't see flask-sqlalchemy.

File: README.md [hunk 3]
```diff
@@ -51,14 +51,21 @@ later:
...
+Nephos has a Web Interface that you can see how everything works. To get started set the enviorment `FLASK_APP` to the path where webServer.py is and run it with `flask run` and you can see the database entries in an instant!
+
 ## Developer Documentation
 Developers can view the documentation that is present for users since it is detailed and one needs to read it in order to understand how Nephos functions. Along with that, docstrings have been placed in HTML format in
 `docs/DevDocs` and can be accessed in a systematic manner by opening `docs/DevDocs/nephos.html` in a browser.
 
 You can also go [here](https://thealphadollar.github.io/NephosDevDocs/) to access it online.
 
+## Tests
+In order for all code to be successful and well maintanable we write run to test out the code. To do so run `pipenv run py.test`
```
Comment (thealphadollar) [inline README.md]: "we run tests" not "we write run"

**Pull request merged**